app/app.js

```diff
@@ -10,20 +9,25 @@ function Chicken (chix)  {
   this.publishedOn = chix.publishedOn;
   this.description = chix.description;
   this.image = chix.image;
+  this.toHtml = function() {
+    this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
+    var template = $('#chickenTemplate').html();
+    var compiledTemplate = Handlebars.compile(template);
+    // $('#chickens').append(compiledTemplate(this));
+    return compiledTemplate(this);
+  };
```

Comment:
While this does create the toHtml function on the Chicken object it is creating it on ALL of them. It's a better practice to have this applied as a prototype function, so only one version of the function is created and all the Chicken objects use it (instead of their own copies of it, as you now have it).